src/foam/nanos/ticket/Ticket.js

```diff
@@ -404,26 +405,34 @@ foam.CLASS({
         if (subject.getUser().getId() == getCreatedFor()) {
           if ( getAssignedTo() != 0 ) {
             Notification notification = new Notification.Builder(x)
-              .setBody(this.COMMENT_NOTIFICATION)
+              .setBody(this.COMMENT_NOTIFICATION + getId())
```

Comment:
Notifications can use notification templates, which can use email templates, both support 'arguments' like the getId(). 

And then we can have a TicketNotificationCitationView that can jump to the ticket with the getId().